derp/derp_server.go

```diff
@@ -510,6 +515,7 @@ func (c *sclient) run(ctx context.Context) error {
 			}
 			return fmt.Errorf("client %x: readFrameHeader: %w", c.key, err)
 		}
+		go c.s.clientBandwidth.Map.Add(c.remoteAddr, int64(fl))
```

Comment:
we can't have this map growing unbounded forever. also: remoteAddr changes not just on IP but also on port.

Reply:
I added a timer so that whenever anything new is added to the map, it removes items that weren't added in a certain time.